### ISSUE TITLE
resemble.js Output Settings working?

### ISSUE THREAD

**Issue by armandsz22** [codeceptjs/codeceptjs-resemblehelper#118]
Hi, is this option resemble.js Output Settings working?
Tried to follow instructions, tried ignoredBox. No effect. I want to use ignoredBoxes. 
    const outputSettings = {
      ignoredBox: {  left: 184, top: 231, right: 284, bottom: 254}
    };
    I.seeVisualDiff("Reporing_TimeCorrection-CorrectionOfDetails.png", {prepareBaseImage: baseImage.desktopPages.reporting.timeCorrection, tolerance: 0.00,  outputSettings:outputSettings})

**Comment by saadichouaib** [comment]
I have the same issue. After debugging I can see the outputSettings object become empty here : 

**Line 81**
```
https://github.com/codeceptjs/codeceptjs-resemblehelper/blob/master/index.js 

if (options.outputSettings) {
        options.outputSettings = {};
      }
```

@puneet0191 do you agree ?

**Comment by saadichouaib** [comment]
@armandsz22 

I just added a fix and it's approved : https://github.com/codeceptjs/codeceptjs-resemblehelper/pull/120

If you want to try it without waiting for next release you can use directly my commit in your project : 

`"codeceptjs-resemblehelper": "git://github.com:codeceptjs/codeceptjs-resemblehelper.git#14d454bcb5b0d0d9be37f2ab1b7540050b896f36",`

**Comment by saadichouaib** [comment]
@puneet0191 we can close this one since its fix was merged #120

**Comment by puneet0191** [comment]
@saadichouaib will close after release, we are testing the new branch right now, will make release next week. cc @sharath2106

**Comment by sharath2106** [comment]
We have released the latest version ([1.9.7](https://www.npmjs.com/package/codeceptjs-resemblehelper)) of the library yesterday. Closing this issue now.
